packages/calcite-components/src/components/action/resources.ts

```diff
@@ -4,12 +4,9 @@ export const CSS = {
   buttonCompact: "button--compact",
   indicatorText: "indicator-text",
   iconContainer: "icon-container",
-  slotContainer: "slot-container",
```

Comment:
Hey @alisonailea I think the changes to this file need to be reverted. There's some build errors occurring right now.

Reply:
This is now fixed (https://github.com/Esri/calcite-design-system/pull/8871).